src/main/java/org/fundacion/jala/converter/view/MainInterface.java

```diff
@@ -70,4 +70,8 @@ public void initInterface() {
         setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
         setVisible(true);
     }
+
```

Comment:
Line 24: You should only import what is necessary, not the entire library "*"

Reply:
It is already done